lighthouse-core/audits/byte-efficiency/uses-webp-images.js

```diff
@@ -21,9 +21,9 @@ class UsesWebPImages extends ByteEfficiencyAudit {
   static get meta() {
     return {
       name: 'uses-webp-images',
-      description: 'Serve images as WebP',
+      description: 'Serve images in next-gen formats',
```

Comment:
I'm wondering if there's a better name than "next-gen". The closest i'm getting is "2010-era". But that doesnt' seem worth it.

Reply:
how about just "better" or "superior"?